lib/qesapdeployment.pm

```diff
@@ -558,7 +558,7 @@ sub qesap_execute {
 
     qesap_execute_conditional_retry(
     cmd => $qesap_script_cmd,
-    error_string => 'Fatal:',
+    error_list => 'Fatal:',
```

Comment:
```suggestion
    error_list => ['Fatal:'],
```

Or whatever is needed to let the used understand it expect a perl list of string